anyrun/src/main.rs

```diff
@@ -178,6 +184,67 @@ struct RuntimeData {
     config_dir: String,
 }
 
+impl RuntimeData {
+    fn state_file(&self) -> String {
+        format!("{}/state.txt", self.config_dir)
+    }
+
+    fn save_state(&self, text: &str) -> io::Result<()> {
+        if !self.config.persist_state {
+            return Ok(());
+        }
+        let timestamp = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_millis();
+        
+        let mut file = fs::File::create(self.state_file())?;
+        writeln!(file, "{}", timestamp)?;
```

Comment:
I'd much prefer if the state file was a struct serialized with ron like all the other config related files, this would be more consistent and allow for further extension if necessary. Something along the lines of:
```
struct PersistentState {
    timestamp: u128,
    text: String,
}
```

Reply:
I first did that, but changed it for a text file, to avoid unnecessary serialization/deserialisation on some simple structure like this one.
But it make sense to keep it !